types/v56/types.go

```diff
@@ -192,7 +192,7 @@ type SubAllocation struct {
 // Description: Represents a list of IP scopes.
 // Since: 5.1
 type IPScopes struct {
-	IPScope IPScope `xml:"IpScope"` // IP scope.
+	IPScope []*IPScope `xml:"IpScope"` // IP scope.
```

Comment:
Please elaborate why this change was needed. It impacted existing code quite a bit with additional casts like these:
`IPScope: []*types.IPScope{&types.IPScope{`

Reply:
Existing implementations allows only one ip scope - it wasn't correct. vCD allows to have many ip scopes.

Reply:
Still looks a disturbing (a lot of existing code) change, but maybe it's only me. @Didainius and @dataclouder what do you think?

Reply:
Well this only fix a bug in govcd. All implementation still if were using one scope [0] still using.

Reply:
Got it, we were not following XML serialisation of vCD API here and now we do. Further issues that became visible after this change will be tracked here:
https://github.com/terraform-providers/terraform-provider-vcd/issues/235